### PR TITLE
New input field implementation

### PR DESCRIPTION
**Pull request by jayrmotta**
This is the new implementation requested by Helen of how our input fields and feedback tags will look like. As you can see we'll have side-effects in a lot of other screens.

screen shot 2018-06-23 at 12 58 50
screen shot 2018-06-23 at 13 00 45
screen shot 2018-06-23 at 13 01 47

Feedbacks are welcome!

### CODE REVIEW

File: src/ui/form/form.js [hunk 1]
```diff
@@ -60,6 +46,16 @@ const FormGroupTag = styled.div`
   text-align: ${props => (props.center ? 'center' : 'left')};
 `;
 
+const InputFormGroupTag = styled.div`
+  text-align: ${props => (props.center ? 'center' : 'left')};
+  box-shadow: 0px 0px 12px 2px #eaf1f3;
```
Comment (v3rm0n) [inline src/ui/form/form.js]: Should we get colors from variables?

Comment (jayrmotta) [reply src/ui/form/form.js]: That's a good point indeed! I'll fix that.. thank you :)

**Comment by jayrmotta** [comment]
@almeynman I've sent a message a couple days ago about this in our slack, in order to push this everyone has to make some adjustments on screens.

Let's discuss this tomorrow, but we gotta have @HelenC on the loop as well.

**Comment by jayrmotta** [comment]
@HelenC @v3rm0n @JordanValdma @ErkoRisthein 

Do we still see any point in merging this into master? Given that we're moving away from web and to material design.

**Comment by almeynman** [comment]
I think yes, these are based on learnings and are done, right?

**Pull request closed without merging**